### PR TITLE
git-reset: expand page

### PR DESCRIPTION
**Pull request by waldyrious**
As mentioned in https://github.com/tldr-pages/tldr/pull/1088#issuecomment-249368792

### CODE REVIEW

**Comment by waldyrious** [comment]
Also requesting review by @sethwoodworth, who [has contributed](https://github.com/tldr-pages/tldr/commits?author=sethwoodworth) to several git subcommand pages.

File: pages/common/git-reset.md [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+
+`git reset -p {{path/to/file}}`
+
+- Undo the last commit, keeping its changes (and any further uncommitted changes) in the filesystem:
+
+`git reset HEAD~`
+
+- Undo the last two commits, adding their changes to the index, i.e. staged for commit:
+
+`git reset --soft HEAD~2`
```
Comment (waldyrious) [inline pages/common/git-reset.md]: After reading http://www.paulboxley.com/blog/2011/06/git-caret-and-tilde I stopped using ^ unless I'm talking about merge commits. The analogy I used was that `~1` means "parent", `~2` means "grandparent", and so on; and that `^1` means "father", and `^2` means "mother". THe bifurcation in the ^ helps me remember this :)

Comment (ostera) [inline pages/common/git-reset.md]: Again, you're right. I wasn't using `HEAD^1`, just `HEAD`).

**Comment by agnivade** [comment]
:+1:

**Comment by waldyrious** [comment]
@ostera so what's your verdict? :)

**Comment by ostera** [comment]
LGTM!

**Pull request merged**